### ISSUE TITLE
Can this method be applied to stable diffusion?

### ISSUE THREAD

**Issue by dingjingzhen** [Hramchenko/diffusion_distiller#3]
Can this method be applied to stable diffusion?

**Comment by KohakuBlueleaf** [comment]
@dingjingzhen Yes, you can just checkout this paper: https://arxiv.org/pdf/2210.03142.pdf.
Basically this method can be applied on any diffusion model.
You can say what this method doing is just train a new model with special noise schedule and target.
And the training target is to "simulate" 2step ddim sampling from teacher model.
